### DISCUSSION TITLE
Is there a example node clustering based node feature?

### DISCUSSION THREAD

**Discussion by tb1over** [pyg-team/pytorch_geometric#3855] [Q&A]
hello, Is there a example node clustering based node feature?

**Comment by rusty1s** [comment]
Can you clarify what you are trying to do? Our autoencoder examples may be a good starting point, see [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/autoencoder.py) and [here](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/argva_node_clustering.py).

**Reply by tb1over** [reply]
thank you for your reply, my datasets have not ground truth, i want to clustering the samples and use it to downstream analysis.
the datasets may have 200 or less samples, but every samples may have over 1000 features.

**Reply by rusty1s** [reply]
Yeah, sounds good. I think the examples I mentioned above are a great starting point. Currently, the expect a single graph as input, but it should be quite straightforward to map them to a multiple graph setting.
